docs/application/dotnet/guides/app-management/application-launcher.md

```diff
@@ -0,0 +1,304 @@
+# Application launcher
+
+This guide explains how to create a basic application launcher. The application launcher is a main application that normally starts after system boot. This app is commonly replaced in the platform adjustments process. It is easy to use the .NET APIs in the application launcher implementation.
+
+## Application launcher functionalities
+
+Every application launcher must be able to:
+ - List installed and runnable applications.
+ - Run selected application by the user.
+
+You can also implement advanced functionalities such as:
+ - grouping installed applications into folders,
+ <!-- TBD: when widgets viewer will be pushed unlock this line - [widgets viewer](./widgets_viewer.md), -->
+ - removing applications,
+ - rearrangement of icons,
+ - viewing notifications,
+
+## Prerequisites
+
+To list the installed applications and launch them, the launcher app must have defined privileges in the project manifest file as:
+
+**Snippet: Application launcher priviledges**
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+  <ui-application appid="NUIApplicationLauncher">
+    <!-- Auto generated manifest properties -->
+  </ui-application>
+  <privileges>
+    <privilege>http://tizen.org/privilege/packagemanager.info</privilege>
+    <privilege>http://tizen.org/privilege/appmanager.launch</privilege>
+  </privileges>
+```
+
+## Application launcher sample code
+
+Following code snippet illustrates how to implement the simple application launcher using:
+ - [NUI](/application/dotnet/api/TizenFX/latest/api/Tizen.NUI.html) for the view implementation.
+ - [Package MAnager](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.PackageManager.html) to gather installed applications,
+- [TizenFX AppControl](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.AppControl.html) to launch an app on touch event
+
+**Figure: NUI application launcher**
+
+![Application launcher](./media/application_launcher_animation.gif)
+
+To use mentioned API's following namespaces have to be included:
+
+```csharp
+using System;
+using System.Collections.Generic;
+
+using Tizen.Applications;
+
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+```
+
+To group application launcher responsibilities three classes in the `NUIApplicationLauncher` namespace are defined which represents main application structure:
```

Comment:
As commented earlier can these sections be either written as ordered steps or unordered steps so that there is content flow.